translation/file_loader.go

```diff
@@ -22,38 +18,25 @@ func NewFileLoader(paths []string) *FileLoader {
 	}
 }
 
-func (f *FileLoader) Load(folder string, locale string) (map[string]map[string]string, error) {
-	translations := make(map[string]map[string]string)
+func (f *FileLoader) Load(locale string, folder string) (map[string]map[string]any, error) {
```

Comment:
I think `map[string]map[string]any` is okay since we know that loaded data from file will be `map[string]any`. And in translator file we are also expecting this.

Reply:
The final structure is `map[string]map[string]map[string]any`, we set the first `string` in the translator file, so why not set the second `string` in the translator file, too? It will be easier to understand the whole logic.

Reply:
Okay got it will change it